cpp11/msbuild/ice.proj

```diff
@@ -7,10 +7,29 @@
 
     <!-- Custom task to download files -->
     <!-- Nuget executable -->
-    <PropertyGroup>
-      <NugetExe>$(MSBuildThisFileDirectory)NuGet-4.9.2.exe</NugetExe>
-      <NugetURL>https://dist.nuget.org/win-x86-commandline/v4.9.2/nuget.exe</NugetURL>
-    </PropertyGroup>
+    <Choose>
```

Comment:
Do other proj files need these updates too?

Reply:
Need to check C#